class/cluster-autoscaler.yml

```diff
@@ -1,5 +1,11 @@
 parameters:
   kapitan:
+    dependencies:
+      - type: helm
+        source: https://kubernetes.github.io/autoscaler
+        version: ${cluster_autoscaler:charts:cluster_autoscaler}
+        chart_name: cluster-autoscaler
+        output_path: dependencies/cluster-autoscaler/helmcharts/cluster-autoscaler
```

Comment:
Make sure to use the chart version in the output path to avoid problems when compiling different versions of the chart

Reply:
Added the version to the end or is there any other recommendation to consider?